data/profiles/default.yaml

```diff
@@ -80,7 +79,7 @@ spec:
             resources:
               requests:
                 cpu: 100m
-                memory: 128Mi
+                memory: 1024Mi
```

Comment:
Is this a bug in upstream? Why requested memory size is bigger than limited memory size?
Should we set it as default resources settings for pilot https://github.com/istio/installer/blob/master/istio-control/istio-discovery/values.yaml#L17-L20

Reply:
This is one of several outstanding diffs that will still need to be addressed. I propose we merge this as is to progress, and start iterating on eliminating the remaining changes incrementally - wdyt?

Reply:
OK for me.